q2_feature_table/plugin_setup.py

```diff
@@ -154,78 +154,67 @@
 
 plugin.methods.register_function(
     function=q2_feature_table.merge,
-    inputs={'table1': FeatureTable[Frequency],
-            'table2': FeatureTable[Frequency]},
+    inputs={'tables': List[FeatureTable[Frequency]]},
     parameters={
         'overlap_method': Str % Choices(q2_feature_table.overlap_methods()),
     },
     outputs=[
         ('merged_table', FeatureTable[Frequency])],
     input_descriptions={
-        'table1': 'The first feature table to be merged.',
-        'table2': 'The second feature table to be merged.',
+        'tables': 'The tables to be merged together.',
```

Comment:
Should we emulate the `metadata` merging language here?

e.g. `The tables to be merged together. This option may be specified multiple times.`, or something along those lines?

The same question applies for the other two merge methods impacted here.

Reply:
That's kind of an implementation detail of the CLI, so I don't know if it makes sense here.

Reply:
👍 I agree, but it does seem hard to make the language general enough in an interface. I am cool either way.

Reply:
What about `Multiple tables to be merged together` or something like that?